src/App.js

```diff
@@ -1,20 +1,81 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import Header from './Header.js';
 import ImageList from './ImageList.js';
 import imageData from './data.js';
+import SelectOptionItem from './SelectOptionItem.js';
 import './bootstrap-reboot.min.css';
 import './App.css';
 import './gallery.css';
 
 export default class App extends Component {
+  state = { selected: null };
+
   render() {
-    // imageData.map(image => <ImageItem creature = {image} />);
+    const imageNodes = imageData
+      .filter(image => {
+        if (!this.state.selected) return true;
+        return image.keyword === this.state.selected;
+      });
+
+    const imageNodesTwo = imageData
+      .filter(image => {
+        if (!this.state.selectedTwo) return true;
```

Comment:
I would also console.log your value from this.state.selectedTwo because it's coming back as a string while your horns data is coming in as a Number. So they won't ever equal when you compare them.

Reply:
Definitely would've missed this, thank you.